documentation/modules/configuring/proc-managing-node-pools-ids.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='proc-managing-node-pools-ids-{context}']
+= (Preview) Assigning IDs to node pools for scaling operations
+
+[role="_abstract"]
+This procedure describes how to use annotations for advanced node ID handling by the Cluster Operator when performing scaling operations on node pools. 
+You specify the node IDs to use, rather than the Cluster Operator using the next ID in sequence.
+Management of node IDs in this way gives greater control.
+
+The steps assume that you are using ZooKeeper for cluster management. 
+The operation is not possible for clusters using KRaft mode.
```

Comment:
This is supported in exactly the same way for ZooKeeper or KRaft. So I think this can be removed.